README.md

```diff
@@ -19,99 +16,91 @@ They feature state-of-the-art functionality and complex UX workflows for data-ri
-Commercial licenses enable us to support a full-time staff of engineers, which is simply not possible through the MIT model.
+We offer commercial licenses to developers who need the most advanced components and features that can't reasonably be maintained by the open-source community alone.
+These licenses make it possible for us to support a full-time staff of engineers.
 
-Rest assured that when we release features commercially, it's only because we believe that you will not find a better MIT-licensed alternative anywhere else.
+Rest assured that when we release features commercially, it's only because we believe you won't find a better MIT-licensed alternative anywhere else.
 
-See the [Pricing](https://mui.com/pricing/) page for a detailed feature comparison.
+See the [Licensing page](https://mui.com/x/introduction/licensing/) for complete details.
 
-## Plans
+### Plans
 
-### Community plan
+#### Community plan
 
-The free version of MUI X is [published under an MIT license](https://www.tldrlegal.com/license/mit-license) and is [free forever](https://mui-org.notion.site/Stewardship-542a2226043d4f4a96dfb429d16cf5bd#20f609acab4441cf9346614119fbbac1).
-This version contains features that we believe are maintainable by contributions from the open-source community.
-
-MIT licensed packages:
+The free Community version of MUI X contains components and features that we believe are maintainable by contributions from the open-source community.
+It's published under an [MIT license](https://www.tldrlegal.com/license/mit-license) and it's [free forever](https://mui-org.notion.site/Stewardship-542a2226043d4f4a96dfb429d16cf5bd#20f609acab4441cf9346614119fbbac1).
 
 - [`@mui/x-data-grid`](https://www.npmjs.com/package/@mui/x-data-grid)
 - [`@mui/x-date-pickers`](https://www.npmjs.com/package/@mui/x-date-pickers)
 - [`@mui/x-charts`](https://www.npmjs.com/package/@mui/x-charts)
 - [`@mui/x-tree-view`](https://www.npmjs.com/package/@mui/x-tree-view)
 
-### Pro plan
-
-The Pro version of MUI X expands on the features of the free version with more advanced capabilities such as multi-filtering, multi-sorting, column resizing and column pinning for the data grid; as well as the date range picker component.
+#### Pro plan
 
-The Pro version is available under a commercial license—visit [the Pricing page](https://mui.com/pricing/) for details.
+MUI X Pro expands on the Community version with more advanced features and functionality.
+The Data Grid Pro comes with multi-filtering, multi-sorting, column resizing, and column pinning; you also gain access to the Date and Time Range Picker components.
 
-Pro packages:
+The Pro version is available under a commercial license—visit the [Pricing page](https://mui.com/pricing/) for details.
 
 - [`@mui/x-data-grid-pro`](https://www.npmjs.com/package/@mui/x-data-grid-pro)
 - [`@mui/x-date-pickers-pro`](https://www.npmjs.com/package/@mui/x-date-pickers-pro)
```

Comment:
We are missing the new pro packages here, they are present on the licensing page. 🙈 
```suggestion
- [`@mui/x-date-pickers-pro`](https://www.npmjs.com/package/@mui/x-date-pickers-pro)
- [`@mui/x-tree-view-pro`](https://www.npmjs.com/package/@mui/x-tree-view-pro)
- [`@mui/x-charts-pro`](https://www.npmjs.com/package/@mui/x-charts-pro)
```